apps/web/playwright/teams.e2e.ts

```diff
@@ -36,53 +36,6 @@ testBothFutureAndLegacyRoutes.describe("Teams A/B tests", (routeVariant) => {
 testBothFutureAndLegacyRoutes.describe("Teams - NonOrg", (routeVariant) => {
   test.afterEach(({ users }) => users.deleteAll());
 
-  test("Team Onboarding Invite Members", async ({ page, users }) => {
-    const user = await users.create(undefined, { hasTeam: true });
-    const { team } = await user.getFirstTeamMembership();
-    const inviteeEmail = `${user.username}+invitee@example.com`;
-
-    await user.apiLogin();
-
-    page.goto(`/settings/teams/${team.id}/onboard-members`);
-    // await page.waitForLoadState("networkidle");
-
-    await test.step("Can add members", async () => {
-      // Click [data-testid="new-member-button"]
-      await page.locator('[data-testid="new-member-button"]').click();
-      // Fill [placeholder="email\@example\.com"]
-      await page.locator('[placeholder="email\\@example\\.com"]').fill(inviteeEmail);
-      // Click [data-testid="invite-new-member-button"]
-      await page.locator('[data-testid="invite-new-member-button"]').click();
-      await expect(page.locator(`li:has-text("${inviteeEmail}")`)).toBeVisible();
-      await expect(page.locator('[data-testid="pending-member-item"]')).toHaveCount(2);
-    });
-
-    await test.step("Can remove members", async () => {
-      const removeMemberButton = page.locator('[data-testid="remove-member-button"]');
-      await removeMemberButton.click();
-      await removeMemberButton.waitFor({ state: "hidden" });
-      await expect(page.locator('[data-testid="pending-member-item"]')).toHaveCount(1);
-      // Cleanup here since this user is created without our fixtures.
-      await prisma.user.delete({ where: { email: inviteeEmail } });
```

Comment:
This is unnecessary; we have a particular test to verify these functionalities

Reply:
@anikdhabal Can you point me to these tests? I see we have tests for orgs but not for teams exclusively.